tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +54,7 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
     # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
+    assert result == False
```

Comment:
Great; another "pythonic" and and human-readable way to express this is `assert not result`